test/integration/mjolnirSetupUtils.ts

```diff
@@ -50,12 +50,12 @@ export async function ensureAliasedRoomExists(client: MatrixClient, alias: strin
 
 async function configureMjolnir() {
     try {
-        await registerUser('mjolnir', 'mjolnir', 'mjolnir', true)
+        await registerUser(config.pantalaimon.username, config.pantalaimon.username, config.pantalaimon.password, true)
```

Comment:
What happens if we don't have Pantalaimon?

Reply:
The same thing is already used [here](https://github.com/matrix-org/mjolnir/blob/main/test/integration/mjolnirSetupUtils.ts#L84) and this was just keeping the setup consistent.

So all of our tests depend on username / password being configured in the Pantalaimon config, even though integration tests don't use Pantalaimon itself. And yes this is pretty terrible :scream_cat:

Reply:
Alternatively, we could make it `'mjolnir', 'mjolnir', 'mjolnir'` everywhere. Wouldn't that be simpler?

If you go ahead with using `config.pantalaimon.*`, could you add a comment in harness.yml?